proto/dydxprotocol/vault/params.proto

```diff
@@ -25,4 +27,14 @@ message Params {
 
   // The duration that a vault's orders are valid for.
   uint32 order_expiration_seconds = 6;
+
+  // The number of quote quantums in quote asset that a vault with no perpetual
+  // positions must have to activate, i.e. if a vault has no perpetual positions
+  // and has strictly less than this amount of quote asset, it will not
+  // activate.
+  bytes activation_threshold_quote_quantums = 7 [
```

Comment:
Did we ever create upgrade handler for `x/vault`? If so do we need to initialize this value as well?

Reply:
yep upgrade handler is created for `x/vault`. this value should be automatically initialized as the upgrade handler [calls `DefaultParams()` to get params to set](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/app/upgrades/v5.0.0/upgrade.go#L310)